css/kb-search.css

```diff
@@ -0,0 +1,119 @@
+.wpseo-kb-search-search-bar {
+	padding-bottom: 15px;
+	border-bottom: 1px solid #ccc;
+}
+
+.wpseo-kb-search-results {
+	overflow: auto;
+}
+
+.wpseo-kb-search-no-results {
+	margin-top: 10px;
+}
+
+.wpseo-kb-search-result {
+	display: block;
+	margin: 0;
+	padding-left: 5px;
+	border-bottom: 1px solid #efefef;
+	color: #000;
+	text-decoration: none;
+}
+
+.wpseo-kb-search-result-link {
+	display: block;
+	cursor: pointer;
+}
+
+.wpseo-kb-search-result-link:hover,
+.wpseo-kb-search-result-link:focus {
+	background-color: rgba(164, 40, 106, 0.1);
+}
+
+.wpseo-kb-search-result-title {
+	margin: 0 !important;
```

Comment:
Please see if you can avoid the use of !important. 
https://specificity.keegan.st can come in handy for the specifity calculation.